src/channel/Channel.h

```diff
@@ -107,6 +107,13 @@ class ChannelBuilder
         return *this;
     }
 
+    Credentials::CertificateKeyId & GetTrustedRootId() const { return *mCaseParameters.mTrustedRootId; }
+    ChannelBuilder & SetTrustedRootId(Credentials::CertificateKeyId * trustedRootId)
+    {
+        mCaseParameters.mTrustedRootId = trustedRootId;
+        return *this;
+    }
+
```

Comment:
TrustedRootIdentifier is no longer present in CASE section of spec. See: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/3390

Reply:
Updated.